libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -980,6 +982,28 @@ GCS_MAVLINK::update(run_cli_fn run_cli)
             if (msg_snoop != NULL) {
                 msg_snoop(&msg);
             }
+            AP_Avoidance *avoidance = AP_Avoidance::instance();
```

Comment:
We could move the decoding of this message into the AP_Avoidance library.  Perhaps move it into a handle_message() function like we've done for some other libraries.  I don't feel strongly about it.  The downside of moving it into AP_Avoidance is that it creates a dependency on mavlink.